autogen.sh

```diff
@@ -10,6 +10,9 @@ else
   echo "Failed to find libtoolize or glibtoolize, please ensure it is installed and accessible via your PATH env variable"
   exit 1
 fi;
+PRODUCT_NAME=${PRODUCT_NAME:-suricata}
+PRODUCT_VERSION=${PRODUCT_VERSION:-7.0.3-dev}
+sed -e "s/@PRODUCT_NAME@/$PRODUCT_NAME/" -e "s/@PRODUCT_VERSION@/$PRODUCT_VERSION/"  configure.ac.in > configure.ac
```

Comment:
I think this sed line is why I don't like this. It creates another input template file and diverges us from a standard autotools project layout which could come with its own issues as many packaging tools are built around this.

Could an integrator not do:
```
export PRODUCT_NAME=myids
export PRODUCT_NAME=9.0.0-myids
sed -i "s/AC_INIT.*/AC_INIT([$PRODUCT_NAME],[$PRODUCT_VERSION])/" configure.ac
```
Its pretty normal in the Linux packaging world to modify these input files as needed as there is no one solution for everyone.  Could integrators not do the same?

Reply:
Good point. I'm going to close this PR